tooling/nargo_fmt/tests/expected/tmp/src/main.nr

```diff
@@ -0,0 +1,79 @@
+// Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
```

Comment:
Is this leftover from dev? I don't see the corresponding input file

Reply:
Odd, I'm not sure how that got in there. I think it must have been from an old PR I tested locally a while back? I'll remove it